Framework/Kernel/inc/MantidKernel/RegistrationHelper.h

```diff
@@ -17,12 +17,13 @@ namespace Kernel {
  * the comma operator can be used to make a call to the factories' subscribe
  * method in the first part.
  */
-class MANTID_KERNEL_DLL RegistrationHelper {
+class MANTID_KERNEL_DLL RegistrationHelper{// clang-format off
 public:
   /** Constructor. Does nothing.
    * @param i :: Takes an int and does nothing with it
    */
   inline RegistrationHelper(int i) { UNUSED_ARG(i); }
-};
+// clang-format on
+}; // namespace Mantid
```

Comment:
```suggestion
};
```
this isn't the end of the `Mantid` namespace